neoexchange/photometrics/gf_movie.py

```diff
@@ -144,35 +144,30 @@ def make_gif(frames, title=None, sort=True, fr=100, init_fr=1000, progress=True,
                 c += 1
             i += 1
 
-    # pull header information from first fits file
-    for file in fits_files:
-        try:
-            obj, rn, site, inst, frame_type = get_header_info(file)
-            break
-        except FileNotFoundError:
-            obj = rn = site = inst = frame_type = None
-            continue
-    if not obj and not rn and not site and not inst:
+    # pull out files that exist
+    good_fits_files = [f for f in fits_files if os.path.exists(f)]
+    base_name_list = [os.path.basename(f) for f in good_fits_files]
+    if not base_name_list:
```

Comment:
Don't think it will make a big/any difference but technically should be a check on the `good_fits_files` - otherwise it's a test on the path to the folder of files not existing. 
Also could you make it an explicit test on the length of the `list` ? I find the `if not <foo>` non-intuitive for `bool`s let alone having `[] == False`...